webui/react/src/hooks/useModal/Workspace/useModalWorkspaceCreate.tsx

```diff
@@ -251,7 +253,9 @@ const useModalWorkspaceCreate = ({ onClose, workspaceID }: Props = {}): ModalHoo
           setWorkspace(response);
         } else {
           const response = await createWorkspace(body);
-          routeToReactUrl(paths.workspaceDetails(response.id));
+          if (response) {
```

Comment:
do we need to consider the case if `response` is `undefined`? or we can ignore the case?

Reply:
If I format it like this, `createWorkspace` could always return an `undefined`
I don't think that our system supports `createWorkspace` throwing/returning an error, I think I need to call handleError inside the store code.

Reply:
ok I removed the try/catch and `handleError` from the store, so `createWorkspace` can return a proper `Promise<Workspace>` and an error will travel up to the previous & visible handleError call.